openshift/templates/db/db-deploy.yaml

```diff
@@ -6,6 +6,65 @@ metadata:
     tags: ${NAME}${SUFFIX}
   name: ${NAME}${SUFFIX}-persistent-template
 objects:
+
+  - kind: NetworkPolicy
+    apiVersion: networking.k8s.io/v1
+    metadata:
+      name: ${NAME}${SUFFIX}
+      labels:
+        name: ${NAME}${SUFFIX}
+        app: ${APP_NAME}${SUFFIX}
+        env: ${TAG_NAME}
+    spec:
+      description: |
+        Allow the agent(s) to access the wallet.
```

Comment:
Nit.: this should read "allow api, msg queue, schema spy, backup" to access the db